ui/command_buffer_model.cpp

```diff
@@ -140,7 +142,11 @@ QVariant CommandBufferModel::headerData(int section, Qt::Orientation orientation
     if (orientation == Qt::Horizontal && role == Qt::DisplayRole)
     {
         if (section < CommandBufferModel::kColumnCount)
-            return QVariant(tr(CommandBufferColumnNames[section]));
+        {
+            // Don't print out ib-level column if m_show_level_column set to false
+            if (m_show_level_column || (section != kColumnIbLevel))
```

Comment:
sorry a bit confused here: should here be (!m_show_level_column || (section != kColumnIbLevel))?

Reply:
This if-statement prints out the column name. And it only prints out column name if m_show_level_column is true. So I'm not sure why your if-statement has a "!m_show_level_column". 

The other thing it checks for is that it only prints column name if the column is not Ib-level column. Otherwise it has to check against m_show_level_column.

Reply:
i see. i was confused because the comment says `Don't print out ib-level column if m_show_level_column set to false`, and I thought the early return here means `don't print out ib-level column` but with m_show_level_column equal to true.

Reply:
Fair point! Updated comment to be less confusing.